Sources/InstantSearchCore/Searcher/Multi/MultiSearcher.swift

```diff
@@ -67,8 +67,7 @@ public class MultiSearcher: AbstractMultiSearcher<AlgoliaMultiSearchService> {
     let searcher = FacetSearcher(client: service.client,
                                  indexName: indexName,
                                  facetName: attribute,
-                                 query: query,
-                                 requestOptions: requestOptions)
```

Comment:
do we still want to pass the option, even though it does nothing?

Reply:
Yeah might as well, could potentially avoid something unforeseen.